.github/quarkusbuilditemdoc.java

```diff
@@ -89,12 +91,17 @@ private String getJavaDoc(JavaDocCapable<?> source) {
         MutableMultimap<String, Pair<Path, JavaClassSource>> multimap = Multimaps.mutable.sortedSet
                 .with(Comparator.comparing(o -> o.getTwo().getName()));
         String line;
+        List<String> keywordList = Arrays.asList("extends BuildItem {", "extends SimpleBuildItem {",
+                "extends MultiBuildItem {", "extends EmptyBuildItem {");
```

Comment:
Actually I meant to move it to a private static final constant. I think you can also remove the `{` from each item too. 
```suggestion
        private static final List<String> KEYWORDS = List.of("extends BuildItem", "extends SimpleBuildItem",
                "extends MultiBuildItem", "extends EmptyBuildItem");
```

Reply:
Actually I included " {" here in purpose. I found that `quarkus/deployment/ExtensionLoader.java` will be picked up as well because of the code on line 273. It includes "extends SimpleBuildItem" but I don't think ExtensionLoader.java should be recognized as a build item? (correct me if I'm wrong)

Reply:
Oh, I didn't know about that, in this case perhaps it's better to keep the `{`, or https://github.com/quarkusio/quarkus/pull/22011#pullrequestreview-826449086, WDYT?